test/fixtures/kibana_test/recipes/default.rb

```diff
@@ -2,6 +2,12 @@
 include_recipe 'elasticsearch::default'
 include_recipe 'kibana_lwrp::install'
 
+# sample plugin installation
+kibana_plugin 'kibana/timelion' do
```

Comment:
I think this should just be 'timelion' since that is the plugin name and directory name, right? 
```
kibana_plugin 'timelion' do
  url 'kibana/timelion'
  action :install
end
```

If we had a resource with the actual plugin name, we might be able to better finagle an uninstall action using that name and looking for a directory by the same name.